corehq/apps/export/models/new.py

```diff
@@ -183,6 +183,7 @@ def create_from_question(cls, question, app_id, app_version, repeats):
     @classmethod
     def merge(cls, one, two):
         item = one
+        item.label = two.label  # always take the newest label
```

Comment:
We do pass in the current schema in first (at least for forms) so it will work but agree with you that it's not exactly clear that `two` should take priority over `one`